src/encoded/viewconfigs/matrix.py

```diff
@@ -76,10 +73,11 @@ def _construct_result_views(self, type_info):
             views.append(self._view_item.summary_report)
         return views
 
-    def _construct_xygroupings(self, query, filters, negative_filters):
+    # pylint: disable=no-self-use
```

Comment:
@caseylitton This is not part of the requirements. However, I added it to silence Codacy (this method out to be a static function as "self" is never used). It is something that needs attention in the future.

Reply:
Let remove this, the codacy issue it not a problem.  If we do want to fix it the remove self from the arguments and make add the staticmethod decorator.

Reply:
I removed the comment to disable pylint warning. Going forward, I will not rebase while a PR is undergoing review.